src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -410,8 +410,8 @@ class DocBaseViewer extends BaseViewer {
     }
 
     handleFindBarClose() {
-        if (this.docEl) {
-            this.docEl.focus(); // Prevent focus from transferring to the root document element
+        if (this.findBarToggleElement) {
```

Comment:
We may also want to check that `.focus` exists.

Reply:
ok. Updated